gallery.css

```diff
@@ -0,0 +1,74 @@
+body {
+    font-family: sans-serif;
+    background: black;  
+    padding-top: 50px;
+  }
+  
+  *{
+    font-family: 'Poppins', sans-serif !important;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
+    -moz-user-select: -moz-none;
+    -o-user-select: none;
+    user-select: none;
+}
+
+
+h1{
+    color: #FFF;
+}
+
+
+/*navbar*/
+
+.navbar{
+background: #222 !important;
+line-height: 2rem !important;
+position: fixed;
+top: 0;
+left: 0;
+right: 0;
+z-index: 9999;
+}
+
+.navbar .navbar-brand{
+color: #FFF !important;
+
```

Comment:
The project name could perhaps use a `margin-left` property.

Reply:
Please explain, are you saying the logo should be on the left? or just padding so it doesn't scrape the side of the screen?

Reply:
There should be some space before the project name.